parts/linux/cloud-init/artifacts/components.json

```diff
@@ -714,7 +714,7 @@
               },
               {
                 "renovateTag": "<DO_NOT_UPDATE>",
-                "latestVersion": "1.6.10",
+                "latestVersion": "1.6.11",
```

Comment:
don't we also want to update this? https://github.com/Azure/AgentBaker/blob/8c79cf5887d37209a083bc52541f197280ae1833/vhdbuilder/prefetch/internal/containerimage/testdata/components.json#L68

Reply:
No, that will be updated automatically by renovate tag.